cores/piduino/WString.h

```diff
@@ -123,5 +123,8 @@ class StringSumHelper : public String
   StringSumHelper(double num) : String(num) {}
 };
 
+typedef char __FlashStringHelper;
+#define PROGMEM
+
```

Comment:
should those be defined here? I think they are defined in avr/progmem.h and while we do not have it, we can create it and not have issues down the line with missing include (had that happen to me quite often on the ESP)
and would add the following define in it as well and maybe more required (have to look in the orginal avr header)
```cpp
#define pgm_read_byte(b) *((uint8_t*)(b))
```